scripts/linux_build.sh

```diff
@@ -382,8 +386,11 @@ function run_install() {
     "gcc-ranlib"
   )
 
-  # update alternatives for gcc and g++ if a debian based distro
-  if [ "$distro" == "debian" ] || [ "$distro" == "ubuntu" ]; then
+  #set gcc version based on distros 
+  if [ "$distro" == "arch" ]; then
+    export CC=gcc-14
+    export CXX=g++-14
```

Comment:
How will this work when using cuda? Last time I checked cuda on arch was using gcc13, but that may have been updated since.